### PR TITLE
Rebranding to OpenEthereum

### PR DESCRIPTION
**Pull request by rakita**
* Legal headers
* Network ID change
* binary file change
* Links to openethereum github

### CODE REVIEW

File: ethcore/src/lib.rs [hunk 1]
```diff
@@ -1,62 +1,22 @@
...
-//!
-//!   # download and build parity
-//!   git clone https://github.com/paritytech/parity-ethereum
-//!   cd parity
-//!   cargo build --release
-//!   ```
-
-// Recursion limit required because of
-// error_chain foreign_links.
-#![recursion_limit = "128"]
```
Comment (adria0) [inline ethcore/src/lib.rs:59]: Is this not still used?

File: Cargo.toml [hunk 2]
```diff
@@ -1,10 +1,13 @@
 [package]
-description = "Parity Ethereum client"
-name = "parity-ethereum"
+description = "OpenEthereum"
+name = "openethereum"
 # NOTE Make sure to update util/version/Cargo.toml as well
-version = "2.5.13"
+version = "3.0.1"
```
Comment (adria0) [inline Cargo.toml:5]: 3.1 ?

File: scripts/docker/alpine/Dockerfile [hunk 3]
```diff
@@ -10,34 +10,37 @@ RUN apk add --no-cache \
...
 # show backtraces
 ENV RUST_BACKTRACE 1
 
+# curl and jq are installed to help create health and readiness checks on Kubernetes
 RUN apk add --no-cache \
   libstdc++ \
   eudev-libs \
-  libgcc
+  libgcc \
+  curl \
```
Comment (adria0) [inline scripts/docker/alpine/Dockerfile:31]: Is this a backport of other issue?

File: ethcore/src/lib.rs (hunk 1 above)
Comment (rakita) [reply ethcore/src/lib.rs:59]: The default value is 128, this is why i omitted it. https://doc.rust-lang.org/reference/attributes/limits.html

**Review by adria0 — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 2 above)
Comment (rakita) [reply Cargo.toml:5]: Not sure what is practice, do we update to 3.1.0 just before release or we are putting the next version after release. I would put 3.1 as last commit before our release, but for version, I don't have a strong opinion.

File: scripts/docker/alpine/Dockerfile (hunk 3 above)
Comment (rakita) [reply scripts/docker/alpine/Dockerfile:31]: I tried clang and it fails to compile kvdb this is why I left gcc. 

For curl it is used by Kubernetes or at least it says that in newest docker file: `# curl and jq are installed to help create health and readiness checks on Kubernetes`

File: ethcore/src/lib.rs (hunk 1 above)
Comment (adria0) [reply ethcore/src/lib.rs:59]: > The default value is 128, this is why i omitted it. https://doc.rust-lang.org/reference/attributes/limits.html

Good catch!

**Pull request merged**